jdaviz/configs/specviz/plugins/unit_conversion/unit_conversion.vue

```diff
@@ -19,22 +19,28 @@
       ></v-select>
     </v-row>
 
+    <v-row  v-if="config == 'cubeviz' && show_translator">
```

Comment:
now that 3.10 is released, let's make this always visible (just remove `show_translator` entirely)

Reply:
This is now removed!